PlanMyTrip/PlanMyTrip/View Controllers/DisplayResultsViewController.m

```diff
@@ -64,51 +71,68 @@ - (void)viewDidLoad {
     if(self.flightStatus == 1){
         self.FirstTitle.text = @"Flight Recomendations";
         self.firstThird = YES;
+        self.firstButton.backgroundColor = [UIColor whiteColor];
+        self.secondButton.backgroundColor = [UIColor whiteColor];
+        self.thirdButton.backgroundColor = [UIColor whiteColor];
+        self.firstFilter.backgroundColor = [UIColor whiteColor];
+        [self.firstFilter setTitle:@"Filter Results" forState:UIControlStateNormal];
         if(self.hotelStatus == 1){
             self.SecondTitle.text = @"Hotel Recomendations";
             self.secondThird = YES;
+            self.fourthButton.backgroundColor = [UIColor whiteColor];
+            self.fifthButton.backgroundColor = [UIColor whiteColor];
```

Comment:
All these buttons look super hard-code-ish. Couldn't we check how many results in each array (flights, hotels, cars) and dynamically set up these buttons in an array or sth?